weed/util/http/http_global_client_util.go

```diff
@@ -418,7 +418,11 @@ func CloseResponse(resp *http.Response) {
 	io.Copy(io.Discard, reader)
 	resp.Body.Close()
 	if reader.BytesRead > 0 {
-		glog.V(1).Infof("response leftover %d bytes", reader.BytesRead)
+		if resp.Request != nil && resp.Request.URL != nil {
```

Comment:
this file has no related changes.

Reply:
yes, this change is unrelated to ACL：printing the URL when it's not empty is intended to aid in troubleshooting if issues arise